bower.json

```diff
@@ -5,10 +5,16 @@
     "dist/rollbar.umd.js"
   ],
   "ignore": [
+    "dist/*.snippet*",
```

Comment:
Thanks for pointing this out. I believe you can use rollbar.min.js from the bower installation but I'm going to re-add the snippet for backwards compatibility.